client/layout/masterbar/item-border-wrapper.jsx

```diff
@@ -0,0 +1,12 @@
+export default function ItemBorderWrapper( { isActive } ) {
```

Comment:
This initially was a wrapper that passed children. The name and classnames seem less fitting now that this is interior to the Item components now.